config/app.config

```diff
@@ -45,6 +45,10 @@ irsa.gator.hostname     = "https://irsa.ipac.caltech.edu"
 most.host = "https://irsasearchops.ipac.caltech.edu/cgi-bin/MOST/nph-most"
 // IRSA Periodogram API
 irsa.gator.service.periodogram.url = "https://irsa.ipac.caltech.edu/cgi-bin/periodogram/nph-periodogram_api"
+// IRSA hips map access: from "file" or from "url" && hips url
+irsa.hips.list.source="url"
+irsa.hips.masterUrl="https://irsadev.ipac.caltech.edu/data/hips/list"
```

Comment:
Properties should defaults to OPS.  Also, when we open-source firefly, we had to remove all reference of internal machine names.  I think that's still a policy.  @robyww @xiuqin ?

Reply:
Yes, that is what we want.

Reply:
remove IRSA HiPS map url base from the config files per discussion.